.changeset/tricky-cars-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sunodo/sdk": patch
```

Comment:
Should be minor?

Reply:
I'm not sure. 

It doesn't breaks user but adds an option functionality.